V3: pnpm run dev results in error

### Current Behavior

```
> skeleton@1.0.0 dev /skeleton
> pnpm --reporter append-only --color "/dev:/"

[36m.[39m [96mdev:docs[39m$  pnpm --filter "@skeletonlabs/docs-next" dev
[36m.[39m [96mdev:packages[39m$ pnpm --recursive --filter "@skeletonlabs/skeleton-*" build:watch
[36m.[39m [96mdev:packages[39m: Scope: 2 of 9 workspace projects
[36m.[39m [96mdev:packages[39m: [36mpackages/skeleton-react[39m [96mbuild:watch[39m$ tsc --project tsconfig.package.json --watch --preserveWatchOutput
[36m.[39m [96mdev:packages[39m: [35mpackages/skeleton-svelte[39m [96mbuild:watch[39m$ svelte-package --watch
[36m.[39m [96mdev:docs[39m: > @skeletonlabs/docs-next@0.0.1 dev /skeleton/sites/next.skeleton.dev
[36m.[39m [96mdev:docs[39m: > astro dev
[36m.[39m [96mdev:packages[39m: [36mpackages/skeleton-react[39m [96mbuild:watch[39m: 6:37:35 PM - Starting compilation in watch mode...
[36m.[39m [96mdev:docs[39m: [34m▶ Astro collects anonymous usage data.[39m
[36m.[39m [96mdev:docs[39m:   This information helps us improve Astro.
[36m.[39m [96mdev:docs[39m:   Run "astro telemetry disable" to opt-out.
[36m.[39m [96mdev:docs[39m:   [36m[4mhttps://astro.build/telemetry[24m[39m
[36m.[39m [96mdev:docs[39m: [2m18:37:37[22m [34m[types][39m Generated [2m1ms[22m
[36m.[39m [96mdev:docs[39m: [34m[vite-plugin-pagefind][39m [1mBuilding site using "pnpm build"...[22m
[36m.[39m [96mdev:packages[39m: [35mpackages/skeleton-svelte[39m [96mbuild:watch[39m: [90md.ts type declaration files for the following files were likely not generated due to the following errors:[39m
[36m.[39m [96mdev:packages[39m: [35mpackages/skeleton-svelte[39m [96mbuild:watch[39m: [90m/skeleton/packages/skeleton-svelte/src/lib/internal/create-context.ts[39m
[36m.[39m [96mdev:packages[39m: [35mpackages/skeleton-svelte[39m [96mbuild:watch[39m: [90m  - The inferred type of 'createContext' references an inaccessible 'unique symbol' type. A type annotation is necessary.[39m
[36m.[39m [96mdev:packages[39m: [36mpackages/skeleton-react[39m [96mbuild:watch[39m: 6:37:38 PM - Found 0 errors. Watching for file changes.
[36m.[39m [96mdev:packages[39m: [35mpackages/skeleton-svelte[39m [96mbuild:watch[39m: [1m[32msrc/lib -> dist[22m[39m
[36m.[39m [96mdev:packages[39m: [35mpackages/skeleton-svelte[39m [96mbuild:watch[39m: Watching src/lib for changes...
[36m.[39m [96mdev:docs[39m: [90mCommand failed: pnpm build[39m
[36m.[39m [96mdev:docs[39m: [90mwarn - Your `content` configuration includes a pattern which looks like it's accidentally matching all of `node_modules` and can cause serious performance issues.[39m
[36m.[39m [96mdev:docs[39m: [90mwarn - Pattern: `../../packages/skeleton-react/**/*.ts`[39m
[36m.[39m [96mdev:docs[39m: [90mwarn - See our documentation for recommendations:[39m
[36m.[39m [96mdev:docs[39m: [90mwarn - https://tailwindcss.com/docs/content-configuration#pattern-recommendations[39m
[36m.[39m [96mdev:docs[39m: [90m18:37:55 [ERROR] [vite] x Build failed in 1.21s[39m
[36m.[39m [96mdev:docs[39m: [90m[vite:css] [postcss] ENOENT: no such file or directory, stat '/skeleton/packages/skeleton-react/node_modules/@skeletonlabs/skeleton/dist/core/index.js'[39m
[36m.[39m [96mdev:docs[39m: [90mfile: /skeleton/node_modules/.pnpm/@astrojs+tailwind@5.1.2_astro@4.16.7_@types+node@22.7.9_rollup@4.24.0_typescript@5.6.3__tailw_zojkbfgw5nkmxryn7k2lim3qna/node_modules/@astrojs/tailwind/base.css:undefined:NaN[39m
[36m.[39m [96mdev:docs[39m: [90m  Stack trace:[39m
[36m.[39m [96mdev:docs[39m: [90m    at Object.statSync (node:fs:1658:25)[39m
[36m.[39m [96mdev:docs[39m: [90m    at resolvedChangedContent (/skeleton/node_modules/.pnpm/tailwindcss@3.4.14_ts-node@10.9.2_@types+node@22.7.9_typescript@5.6.3_/node_modules/tailwindcss/lib/lib/content.js:147:42)[39m
[36m.[39m [96mdev:docs[39m: [90m    at /skeleton/node_modules/.pnpm/tailwindcss@3.4.14_ts-node@10.9.2_@types+node@22.7.9_typescript@5.6.3_/node_modules/tailwindcss/lib/processTailwindFeatures.js:46:11[39m
[36m.[39m [96mdev:docs[39m: [90m    at LazyResult.runOnRoot (/skeleton/node_modules/.pnpm/postcss@8.4.47/node_modules/postcss/lib/lazy-result.js:329:16)[39m
[36m.[39m [96mdev:docs[39m: [90m    at LazyResult.async (/skeleton/node_modules/.pnpm/postcss@8.4.47/node_modules/postcss/lib/lazy-result.js:160:30)[39m
[36m.[39m [96mdev:docs[39m: [90m  [1mHint:[22m[39m
[36m.[39m [96mdev:docs[39m: [90m[33m    Browser APIs are not available on the server.[39m
[36m.[39m [96mdev:docs[39m: [90m    [39m
[36m.[39m [96mdev:docs[39m: [90m    If the code is in a framework component, try to access these objects after rendering using lifecycle methods or use a `client:only` directive to make the component exclusively run on the client.[39m
[36m.[39m [96mdev:docs[39m: [90m    [39m
[36m.[39m [96mdev:docs[39m: [90m    See [4mhttps://docs.astro.build/en/guides/troubleshooting/#document-or-window-is-not-defined[24m for more information.[39m
[36m.[39m [96mdev:docs[39m: [90m    [39m[90m[39m
[36m.[39m [96mdev:docs[39m: [90m  [1mStack trace:[22m[39m
[36m.[39m [96mdev:docs[39m: [90m[2m    at Object.statSync (node:fs:1658:25)[39m
[36m.[39m [96mdev:docs[39m: [90m    at /skeleton/node_modules/.pnpm/tailwindcss@3.4.14_ts-node@10.9.2_@types+node@22.7.9_typescript@5.6.3_/node_modules/tailwindcss/lib/processTailwindFeatures.js:46:11[39m
[36m.[39m [96mdev:docs[39m: [90m    at LazyResult.async (/skeleton/node_modules/.pnpm/postcss@8.4.47/node_modules/postcss/lib/lazy-result.js:160:30)[39m
[36m.[39m [96mdev:docs[39m: [90m    at genericNodeError (node:internal/errors:983:15)[39m
[36m.[39m [96mdev:docs[39m: [90m    at ChildProcess.exithandler (node:child_process:419:12)[39m
[36m.[39m [96mdev:docs[39m: [90m    at maybeClose (node:internal/child_process:1104:16)[22m[39m
[36m.[39m [96mdev:docs[39m: /skeleton/sites/next.skeleton.dev:
[36m.[39m [96mdev:docs[39m: [41m[30mERR_PNPM_RECURSIVE_RUN_FIRST_FAIL[39m[49m [31m@skeletonlabs/docs-next@0.0.1 dev: `astro dev`[39m
[36m.[39m [96mdev:docs[39m: [31mExit status 1[39m
[36m.[39m [96mdev:docs[39m: Failed
[41m[30mELIFECYCLE[39m[49m [31mCommand failed with exit code 1.[39m
ELIFECYCLE Command failed with exit code 1.
```

### Expected Behavior

The build should not result in an error.

### Steps To Reproduce

```dockerfile
FROM node:22

RUN npm install -g pnpm

RUN apt-get update && apt-get install -y git && \
    apt-get clean && rm -rf /var/lib/apt/lists/*

RUN git clone https://github.com/skeletonlabs/skeleton.git

WORKDIR /skeleton

RUN git checkout next

RUN pnpm install

CMD ["pnpm", "dev"]
```

### Link to Reproduction / Stackblitz

_No response_

### More Information

_No response_

--- a/sites/next.skeleton.dev/tailwind.config.js
+++ b/sites/next.skeleton.dev/tailwind.config.js
@@ -14,8 +14,8 @@ export default {
 	darkMode: 'class',
 	content: [
 		'./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}',
-		join(__dirname, 'node_modules/@skeletonlabs/skeleton-react/**/*.{html,js,ts,jsx,tsx}'),
-		join(__dirname, 'node_modules/@skeletonlabs/skeleton-svelte/**/*.{html,js,ts,svelte}')
+		join(__dirname, 'node_modules/@skeletonlabs/skeleton-react/dist/**/*.{html,js,ts,jsx,tsx}'),
+		join(__dirname, 'node_modules/@skeletonlabs/skeleton-svelte/dist/**/*.{html,js,ts,svelte}')
 	],
 	theme: {
 		extend: {}

